src/components/dashboard.js

```diff
@@ -1,19 +1,23 @@
-import React, { Fragment, Component } from 'react';
+import React, { Component, Fragment } from 'react';
 import { connect } from 'react-redux';
-import { withRouter, Redirect } from 'react-router-dom';
-
+import { Redirect } from 'react-router-dom';
+import Lists from './dashboard-lists';
```

Comment:
Generally, if you're naming something `dashboard-lists` your import should be `DashboardLists`